cmd/controller-manager/main.go

```diff
@@ -23,6 +23,8 @@ import (
 	"reflect"
 	"syscall"
 
+	"k8s.io/client-go/tools/clientcmd"
+
```

Comment:
Please remove this empty line.

Reply:
external imports and internal imports should have blank. so not only remove the empty line can solve.